backends-velox/src/test/scala/org/apache/gluten/execution/VeloxStringFunctionsSuite.scala

```diff
@@ -519,12 +519,11 @@ class VeloxStringFunctionsSuite extends VeloxWholeStageTransformerSuite {
         s"from $LINEITEM_TABLE limit 5") { _ => }
   }
 
-  ignore("split") {
+  test("split") {
     runQueryAndCompare(
```

Comment:
please also add check the op is transformed. 

Line 121 in [2e797f3](https://github.com/apache/incubator-gluten/commit/2e797f3fb445af53f858f31cd2ae19082a5e4a07)

Reply:
Updated. Thanks.